overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -99,8 +103,21 @@ func (c *refreshCommand) Execute(args []string) error {
 		return fmt.Errorf("can only be used from gate-auto-refresh hook")
 	}
 
-	if c.Proceed && c.Hold {
-		return fmt.Errorf("cannot use --proceed and --hold together")
+	var which string
+	for _, opt := range []struct {
+		val  bool
+		name string
+	}{
+		{c.PrintInhibitLock, "--show-inhibit-lock"},
+		{c.Hold, "--hold"},
+		{c.Proceed, "--proceed"},
+	} {
```

Comment:
nitpick: I would find it more readable if you declared the struct before the `for` loop.

Reply:
We use the approach of unnamed structs in loops in other places, IMHO it is good for such scenarios as it saves on introducing names that are only used once (also, it saves on bikeshedding over such names ;))

Reply:
Ah! But it's sad, that we cannot bikeshed on names ;-)